lib/matplotlib/tests/test_font_manager.py

```diff
@@ -24,6 +24,7 @@ def test_font_priority():
 
 def test_json_serialization():
     with tempfile.NamedTemporaryFile() as temp:
+        temp.close()
```

Comment:
I don't really care how to resolve this, but we should confirm that this doesn't leave a file hanging around on disk, which is what I think this change does.